### DISCUSSION TITLE
[question] Can you elaborate a little more on IDM's privacy danger?

### DISCUSSION THREAD

**Discussion by Shituation** [pluja/awesome-privacy#248] [Q&A] [answered]
I read the recommended apps and IDM is being put into the non-privacy respecting apps.
You may not like it because its commercial, but I think that's up to the user. What I'm interested in is in your information about it being a privacy danger.
What do you know about IDM gathering personal info when used? do you have any proof of this? do they know what you download? 
I read the privacy contract of them and they say they don't get personal info while using the app, but they do get data when you use their website (as 99% of everything on internet).

Could you elaborate a little more on how is this program a privacy risk?

Thank you

**Comment by pluja** [comment, marked as answer]
Hey, I will remove IDM from the "avoid line". Just read their Privacy Policy and seems good.

Download managers were added with this pull request: https://github.com/pluja/awesome-privacy/pull/147

To be fair, I did not see the line about IDM, it sneaked up on me and I didn't check it.
